Azure/azure-cli
# [Storage] `az storage account create/update`: Support sas&key policy

**Description**

Support set `KeyPolicy` and `SASPolicy` when creating or updating storage account

**Testing Guide**

`az storage create/update -n mystorageaccount -g myresourcegroup --key-exp-days 3 --sas-exp 5.23:59:59`

---

This checklist is used to make sure that common guidelines for a pull request are followed.
